src/index.html

```diff
@@ -115,6 +118,54 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
           collaboration, and confidence.</h5>
       </section>
 
+      <section id="reading-challenge" class="bg-pink col-12 py-5">
+        <div class="text-center col-lg-7 col-xs-8 mx-auto mb-3">
+          <a href="https://lincolnlibraries.org/" id="LincolnCityLibraries" target="_blank" rel="noopener">
+            <img class="logo library" src="/assets/img/partners/lincolnlibraries.png" alt="Lincoln City Libraries">
+          </a>
+          <h2 class="font-weight-bold">2021 STEM Reading Challenge</h2>
+          <i>
+            April 1, 2021 - May 15, 2021
+          </i>
+        </div>
+
+        <div class="col-12 d-flex justify-content-center">
+          <div class="col-lg-8 col-xs-10">
+            <h3 class="text-center mb-4">Join Girls Code Lincoln and Lincoln City Libraries for a STEM Reading Challenge!</h3>
+            <p >
+              By taking part in this challenge, children can learn about the rich history and
+              current contributions of women in Science, Technology, Engineering and Math (S.T.E.M.).
+              This challenge is open to children of all ages in Lincoln.
+            </p>
+            <p>
+              To get started, sign up on the <a href="https://lincolnlibraries.beanstack.org/" target="_blank">Beanstack site</a> or app.
+              To complete the challenge, read four books from our <a href="https://lincolnlibraries.beanstack.org/lists/20851" target="_blank">booklist</a>
+              or choose any book about women in STEM and write reviews.
+            </p>
+            <p>
+              Stop by a library location to pick up an Empowerment bookmark! Everyone who registers and completes the challenge
```

Comment:
ooo should we add some pictures to this section such as a picture of the empowerment bookmark?

Reply:
that's a good idea! maybe I'll look into adding that in the future